tirtha_bk/static/js/main.js

```diff
@@ -640,21 +655,21 @@ function preventDefaults (e) {
     doc.body.addEventListener(evt, preventDefaults, false)
 })
 
-// Highlights drop area, when item is dragged over it
-;['dragenter', 'dragover'].forEach(evt => {
-    upGal.addEventListener(evt, () => {
-        upGal.classList.add('highlight');
-    }, false);
-})
-;['dragleave', 'drop'].forEach(evt => {
-    upGal.addEventListener(evt, () => {
-        upGal.classList.remove('highlight');
-    }, false);
-})
+    // Highlights drop area, when item is dragged over it
```

Comment:
From this line (658) to L668, remove the initial tab space (i.e., revert these lines back to how they were).